src/protocol/index.js

```diff
@@ -154,4 +159,20 @@ message PSResponse {
   repeated string topics = 1;
   repeated bytes peerIDs = 2;
 }
+
+message PeerstoreRequest {
+  enum Type {
+    GET_PROTOCOLS = 1;
+    GET_PEER_INFO = 2;
```

Comment:
Yeah I'll create an issue for when the spec gets done. I only added these initial two, as those likely won't change, but since the spec is still being discussed it didn't make sense to me to pull in a bunch of theoretical types.

I'll add `ERR_NOT_IMPLEMENTED` for the types that are currently in the proto file.